src/dsp_tools/utils/xml_upload.py

```diff
@@ -422,20 +423,29 @@ def _check_consistency_with_ontology(
     if verbose:
         print("Check if the resource types and properties are consistent with the ontology...")
         logger.info("Check if the resource types and properties are consistent with the ontology...")
-    if not any(x.startswith(ontoname) for x in resclass_name_2_type.keys()):
-        err_msg = (
-            f"The <knora> tag of your XML file references the ontology '{ontoname}', "
-            f"but the project {shortcode} on the DSP server doesn't contain an ontology with this name."
-        )
-        logger.error(err_msg)
-        raise UserError(err_msg)
```

Comment:
This entire block is a duplicate that is already contained in `_check_if_onto_name_exists()`.